images/custom-error-pages/rootfs/main.go

```diff
@@ -68,6 +69,17 @@ const (
 	// client does not specify an Accept header, or the Accept header provided
 	// cannot be mapped to a file extension.
 	DefaultFormatVar = "DEFAULT_RESPONSE_FORMAT"
+
+	// IsExportMetricsVar is the name of the environment variable indicating
+	// whether or not to export /metrics and /debug/vars.
+	IsExportMetricsVar = "IS_EXPORT_METRICS"
```

Comment:
I think this name can be modified.  `ExportMetricsEnvVar` or  `IsMetricsExportEnvVar` ?

Reply:
Should be resolved with 7bba591ea79485531671c01226507018b5a7a05e

Please take another look